Stripe/PublicHeaders/STPShippingMethod.h

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
We're going to drop support for iOS 7 in version 9.0, so if this branch is going to come after that it's safe to not make classes like this and just use PKShippingMethod etc.